disnake/types/onboarding.py

```diff
@@ -6,6 +6,7 @@
 from .snowflake import Snowflake, SnowflakeList
 
 OnboardingPromptType = Literal[0, 1]
+OnboardingMode = Literal[0, 1]
 
 
 class OnboardingPromptOption(TypedDict):
```

Comment:
somewhat related, `emoji` here should be a `PartialEmoji`

Reply:
`PartialEmoji` doesn't include `animated` 🤔🤔

Reply:
Yeah, though this is how it's done for components and reactions as well, which also may or may not have an `animated` field; however we're definitely not getting a full `Emoji` object here :/

Reply:
Makes sense 👍

Reply:
b188bd515dd8b67cd6e58830c7690a8ae38d2e44